llvm/lib/Bitcode/Reader/BitcodeReader.cpp

```diff
@@ -858,6 +859,14 @@ class BitcodeReader : public BitcodeReaderBase, public GVMaterializer {
     }
   }
 
+  Expected<ConstantRange>
+  readBitWidthAndConstantRange(ArrayRef<uint64_t> Record, unsigned &OpNum) {
+    if (Record.size() - OpNum < 3)
```

Comment:
I'd check `1` instead of `3` and let `readConstantRange` deal with its own reading error

Reply:
Done.